aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -196,11 +201,11 @@ def _add_cors(self, cors: CORSConfig):
 
     def _add_cache_control(self, cache_control: str):
         """Set the specified cache control headers for 200 http responses. For non-200 `no-cache` is used."""
-        self.response.headers["Cache-Control"] = cache_control if self.response.status_code == 200 else "no-cache"
+        self.response.headers["Cache-Control"].append(cache_control if self.response.status_code == 200 else "no-cache")
```

Comment:
```suggestion
        cache_control = cache_control if self.response.status_code == 200 else "no-cache"
        self.response.headers["Cache-Control"].append(cache_control)
```

np: that's quite a long line to follow what exactly it's being appended. Making a separate var makes it easier to follow.